### DISCUSSION TITLE
how does password-file work

### DISCUSSION THREAD

**Discussion by tshepang** [rustic-rs/rustic#246] [Q&A] [answered]
I saw `password-file` in [an example](https://github.com/rustic-rs/rustic/blob/main/examples/local.toml), but it did not work when I tried it (by placing a password in that file, but still got prompted to input a password)

**Comment by aawsome** [comment, marked as answer]
Is this the `init` command and are you using the release version or the latest beta?
There was a bug which is fixed (#224) but not yet included in a released version.

**Reply by tshepang** [reply]
I use the release version
